ui/views.py

```diff
@@ -57,7 +57,8 @@ def get(self, request, *args, **kwargs):
             "name": name,
             "username": username,
             "host": webpack_dev_server_host(request),
-            "edx_base_url": settings.EDXORG_BASE_URL
+            "edx_base_url": settings.EDXORG_BASE_URL,
+            "search_url": "http://localhost:9200",
```

Comment:
Oh yeah I forgot about that. The same is actually true on linux as well, the `ELASTICSEARCH_URL` environment variable we set for the web container is just for communication between the docker containers. The change I made above to `docker-compose.yml` adds port forwarding so that `localhost:9200` is forwarded to the docker container.

I think we don't have to use that port though. Wasn't the problem that devstack also has an elasticsearch API running at `localhost:9200`?